### PR TITLE
Adds ML Director Insights Part 3

### PR DESCRIPTION
**Pull request by BritneyMuller**
This PR adds ML Director Insights [Part 3: Finance Edition] to the blog. 

Screen Shot 2022-06-14 at 9 15 44 AM
Shot 2022-06-07 at 4 54 56 PM
Shot 2022-06-07 at 4 54 58 PM

### CODE REVIEW

File: ml-director-insights-3.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        <div class="bfc">
+            <code>britneymuller</code>
+            <span class="fullname">Britney Muller</span>
+        </div>
+    </a>
+</div>
+</head>
+<body>
+
+_If you're' interested in building ML solutions faster visit [hf.co/support](https://huggingface.co/support?utm_source=article&utm_medium=blog&utm_campaign=ml_director_insights_3) today!_
```
Comment (feconroses) [inline ml-director-insights-3.md:31]:
```suggestion
_If you're interested in building ML solutions faster visit [hf.co/support](https://huggingface.co/support?utm_source=article&utm_medium=blog&utm_campaign=ml_director_insights_3) today!_
```

File: ml-director-insights-3.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+
+**Fun Fact:** Debanjan played cycle polo at the world's oldest polo club (the Calcutta Polo Club) when he was a kid.
+ 
+**Moody's Analytics:** Provides financial intelligence and analytical tools supporting our clients’ growth, efficiency and risk management objectives.
+
+#### **1. How has ML made a positive impact on finance?**
+Machine learning has made a huge positive impact on finance, is helping combat financial crimes, and makes the world a safer place. One of the first applications ML was used for in finance was fraud detection. 
+
+Working in regtech (the management of regulatory processes within the financial industry through technology),, Machine learning has been a game-changer and a crucial tool in many applications such as Know Your Customer (KYC) screening and Anti Money Laundering (AML). With an increasing spend in AML fines by financial institutions worldwide, the ever-changing realm of sanctions, and increasing complexity in fighting money laundering, banks are making huge investments in KYC and AML technologies powered by ML.
```
Comment (feconroses) [inline ml-director-insights-3.md:108]:
```suggestion
Working in regtech (the management of regulatory processes within the financial industry through technology), machine learning has been a game-changer and a crucial tool in many applications such as Know Your Customer (KYC) screening and Anti Money Laundering (AML). With an increasing spend in AML fines by financial institutions worldwide, the ever-changing realm of sanctions, and increasing complexity in fighting money laundering, banks are making huge investments in KYC and AML technologies powered by ML. 
```

File: ml-director-insights-3.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+**U.S. Bank:** The fifth-largest commercial bank in the United States, U.S. Bank blends its relationship teams, branches and ATM networks with digital tools that allow customers to bank when, where and how they prefer. 
+
+#### **1. How has ML made a positive impact on finance**
+Machine learning and artificial intelligence have made a profound and positive impact on finance in general and banking in particular. There are many applications in banking where many factors (features) are to be considered when making a decision and ML has traditionally helped in this respect. For example, the credit score we all universally rely on is derived from a machine learning algorithm. 
+ 
+Over the years ML has interestingly also helped remove human bias from decisions and provided a consistent algorithmic approach to decisions. For example, in credit card/loan underwriting and mortgages, modern AI techniques can take more factors (free form text, behavioral trends, social and financial interactions) into account for decisions while also detecting fraud.
+
+
+#### **2. What are the biggest ML challenges within SaaS?**
```
Comment (feconroses) [inline ml-director-insights-3.md:158]:
```suggestion
#### **2. What are the biggest ML challenges within Finance?**
```

File: ml-director-insights-3.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+The finance and banking industry brings a lot of challenges due to the nature of the industry. First of all, it is a highly regulated industry with government oversight in many aspects. The data that is often used is very personal and identifiable data (social security numbers, bank statements, tax records, etc). Hence there is a lot of care taken to create machine learning and AI models that are private and unbiased. Many government regulations require any models to be explainable. For example, if a loan is denied, there is a fundamental need to explain why it is denied. 
+ 
+The data on the other hand, which may be scarce in other industries is abundant in the financial industry. (Mortgage records have to be kept for 30 years for example). The current trend for digitization of data and the explosion of more sophisticated AI/ML techniques has created a unique opportunity for the application of these advances.
+
+
+#### **3. What’s a common mistake you see people make trying to integrate ML into SaaS?**
```
Comment (feconroses) [inline ml-director-insights-3.md:164]:
```suggestion
#### **3. What’s a common mistake you see people make trying to integrate ML into finance?**
```

**Review by feconroses — APPROVED** [review]
my favorite piece of the series! I really _really_ liked this, great job! Each of the interviewees provided interesting answers + industry-specific insights 👏

PR looks good to me. Some high level comments/suggestions:
- Curious how the first EAP CTA worked in edition 2 vs edition 1? was it an improvement? Is the new placement (at the top) working? Or should we experiment with something different on this third edition?
- Personally, I'm not a fan of using unnecessary capitalization (e.g. "back to our Director of ML Insights Series..."
- Feel that we could mention the companies' names in the intro. Royal Bank, Moody's & US Bank are all awesome institutions and it may spark more interest to keep reading.
- BTW can we mention Bloomberg here in this post?
- I'll start using the phrase "_Buckle up Goose_" quite often (loved it)

**Comment by BritneyMuller** [comment]
> Curious how the first EAP CTA performed in edition 2 vs edition 1? was it an improvement? Is the new placement (at the top) working? Or should we experiment with something different on this third edition?

Unfortunately ga data doesn't show very clearly which one sent over the most traffic because the more nuanced tracking at the end of the tag is not being picked up in ga for some reason. :\ Let me know if you know of any workarounds for this. I've tried everything in the Campagin section.

> BTW can we mention Bloomberg here in this post?

They aren't really featured, but if you have a clever way to do this, let me know! :) 

- Will add the other Companies included in the intro
- Can fix the capitalization that you hate 😂 #LibraProblems
- Glad you like the "Buckle up, Goose"/Top Gun reference lol. My first draft was even heavier into the play on Top Gun but I figured it probably wouldn't age well over time so I removed most of it.

**Comment by BritneyMuller** [comment]
Have all your edits + Director's edits ready to go. 

Simply waiting on U.S. Bank's edit of their blurb & to get a 👍 from Ioannis at RBC. 🎉

**Review by osanseviero — APPROVED** [review]
Looks good! I just did a light reading, but it's 🔥 🔥

**Comment by feconroses** [comment]
> > BTW can we mention Bloomberg here in this post?
> 
> They aren't really featured, but if you have a clever way to do this, let me know! :)

@BritneyMuller they aren't really featured, but they are mentioned. Given the recent events, I'm wondering if we can still do this? 

Screen Shot 2022-06-10 at 11 33 35 AM

**Comment by BritneyMuller** [comment]
> > > BTW can we mention Bloomberg here in this post?
> > 
> > 
> > They aren't really featured, but if you have a clever way to do this, let me know! :)
> 
> @BritneyMuller they aren't really featured, but they are mentioned. Given the recent events, I'm wondering if we can still do this?
> 
> Screen Shot 2022-06-10 at 11 33 35 AM

:) I know. —It just reads weird to say ex-Bloomberg AI Scientist and I feel Debanjan's title speaks for itself?

**Comment by feconroses** [comment]
> > > > BTW can we mention Bloomberg here in this post?
> > > 
> > > 
> > > They aren't really featured, but if you have a clever way to do this, let me know! :)
> > 
> > 
> > @BritneyMuller they aren't really featured, but they are mentioned. Given the recent events, I'm wondering if we can still do this?
> > Screen Shot 2022-06-10 at 11 33 35 AM
> 
> :) I know. —It just reads weird to say ex-Bloomberg AI Scientist and I feel Debanjan's title speaks for itself?

Yeah, IMO `Director of AI/ML at Moody's` is still good. But do what you think is better!

**Comment by BritneyMuller** [comment]
> > > BTW can we mention Bloomberg here in this post?
> > 

Does this work:
Screen Shot 2022-06-14 at 9 30 27 AM

**Pull request merged**